test/extended/apiserver/apply.go

```diff
@@ -65,7 +65,7 @@ var _ = g.Describe("[sig-api-machinery][Feature:ServerSideApply] Server-Side App
 			continue
 		}
 
-		g.It(fmt.Sprintf("should work for %s", gvr), func() {
+		g.It(fmt.Sprintf("should work for %s [apigroup:%s]", gvr, gvr.Group), func() {
```

Comment:
I don't think this will work. All the apigroup labels are assumed to be static and known in advance. The test is required to run to evaluate all available possibilities. Which is not the case when running openshift-tests binary in dry mode. Checking all apigroups under https://github.com/pacevedom/origin/blob/5619156de146c255b91ab1240695c003b0b685ae/test/extended/etcd/etcd_storage_path.go#L39 it seems none of them is served in MicroShift. So the entire test can be skipped in MicroShift installation is present. Nevertheless, I don't see any obvious generic solution for static apigroup labeling.

Reply:
Just executed it:
```
$ KUBECONFIG=~/.kube/microshift ./openshift-tests run --dry-run openshift/conformance | grep "Server-Side Apply should work for" > /tmp/tests.txt

$ head -5 /tmp/tests.txt 
"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for apps.openshift.io/v1, Resource=deploymentconfigs [apigroup:apps.openshift.io] [Suite:openshift/conformance/parallel]"
"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for build.openshift.io/v1, Resource=buildconfigs [apigroup:build.openshift.io] [Suite:openshift/conformance/parallel]"
"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for build.openshift.io/v1, Resource=builds [apigroup:build.openshift.io] [Suite:openshift/conformance/parallel]"
"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for image.openshift.io/v1, Resource=images [apigroup:image.openshift.io] [Suite:openshift/conformance/parallel]"
"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for image.openshift.io/v1, Resource=imagestreams [apigroup:image.openshift.io] [Suite:openshift/conformance/parallel]"

$ wc -l /tmp/tests.txt 
17 /tmp/tests.txt

$ KUBECONFIG=~/.kube/microshift ./openshift-tests run --provider none -f /tmp/tests.txt --junit-dir /tmp/tests
...
started: (0/1/17) "[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for route.openshift.io/v1, Resource=routes [apigroup:route.openshift.io] [Suite:openshift/conformance/parallel]"
...
passed: (2.3s) 2022-09-21T14:33:27 "[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for route.openshift.io/v1, Resource=routes [apigroup:route.openshift.io] [Suite:openshift/conformance/parallel]"
...
Sep 21 14:33:27.618 I e2e-test/"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for route.openshift.io/v1, Resource=routes [apigroup:route.openshift.io] [Suite:openshift/conformance/parallel]" finishedStatus/Passed
...
1 pass, 16 skip (2.3s)
```

The skipped tests show as
```
Sep 21 14:33:25.337 - 2s    W clusteroperator/[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for template.openshift.io/v1, Resource=templateinstances [apigroup:template.openshift.io] [Suite:openshift/conformance/parallel] e2e test did not finish "DidNotFinish"
Sep 21 14:33:25.337 I e2e-test/"[sig-api-machinery][Feature:ServerSideApply] Server-Side Apply should work for template.openshift.io/v1, Resource=templateinstances [apigroup:template.openshift.io] [Suite:openshift/conformance/parallel]" started
...
```